master/docs/relnotes/0.9.0b2.rst

```diff
@@ -0,0 +1,89 @@
+Release Notes for Buildbot 0.9.0b2
+==================================
+
+..
+    Any change that adds a feature or fixes a bug should have an entry here.
+    Most simply need an additional bulleted list item, but more significant
+    changes can be given a subsection of their own.
+
+The following are the release notes for Buildbot 0.9.0b2.
+Buildbot 0.9.0b2 was released on the 02nd of August, 2015.
+
+Master
+------
+
+Features
+~~~~~~~~
+
+* Mercurial hook was updated and modernized.
+  It is no longer necessary to fork.
+  You can now extend PYTHONPATH via the hook config which allows you to use a buildbot-venv instead of installing buildbot to the system site-packages.
```

Comment:
One can now extend PYTHONPATH via the hook configuration. Among others, it permits to use a buildbot virtualenv instead of installing buildbot in all the system.